.travis.yml

```diff
@@ -17,15 +25,11 @@ script:
     - phantomjs Resources/js/run-qunit.js file://`pwd`/Resources/js/index-with-es5-shim.html
 
 matrix:
-    allow_failures:
-        - env: SYMFONY_VERSION=dev-master
-        - php: hhvm
+    fast_finish: true
     include:
+      - php: 5.3.3
+        env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable'
```

Comment:
those options are available on the `update` command only anyway, so removing them.